feat: isolated environment for getting started

Convolutio opened the pull request:
a conda environment is available with quick installation instructions in the README to build and develop in the project

NemoDacremont commented on environment.yml:
```diff
@@ -0,0 +1,106 @@
+name: /home/thormas/Documents/ENSEIRB/Eirbware/com.eirb.fr/com-env
```
idk much about conda but it looks sus ;)

Convolutio replied:
Oops this is sus

NemoDacremont commented on environment.yml:
```diff
@@ -0,0 +1,106 @@
+  - virtualenv=20.32.0=pyhd8ed1ab_0
+  - zipp=3.23.0=pyhd8ed1ab_0
+  - zstandard=0.23.0=py313h536fd9c_2
+prefix: /home/thormas/Documents/ENSEIRB/Eirbware/com.eirb.fr/com-env
```
same

Convolutio replied:
It's not thought for security and production, just for quickly installing the required development environment (with Python and Bunjs)

Convolutio replied:
But I have to admit that this file is sus. I will find a better way for a quick installation with conda

Convolutio replied:
Ok I should have read [this very basic section of the doc](https://docs.conda.io/projects/conda/en/stable/user-guide/tasks/manage-environments.html#building-identical-conda-environment). I have exported the environment in a very dirty way

Convolutio committed:
feat: isolated dev conda env + support in just

to help the developer with a quick and isolated installation,
a conda environment is available

Convolutio commented:
Normally, it is clean with this setup.

bourgouinraphael reviewed (approved):
idk much about it but it lgtm
